demo.py

```diff
@@ -59,19 +54,24 @@ def print_sep(char='-'):
         print 'Checking status of pairing request...'
         
         try:
-            pairing_status = api.get_pairing_status(pairing_id)
-            if pairing_status.enabled:
+            pairing = api.get_pairing_by_id(pairing_id)
```

Comment:
Do we have a method like `Pairing.refresh()` or `Pairing.sync()` that will get the pairing's up-to-date information?